### ISSUE TITLE
Errors on latest irc mod and server.

### ISSUE THREAD

**Issue by fyregryph** [ShadowNinja/minetest-irc_commands#2]
On my server running minetest 4.11 I get the following error when I try to login through IRC;

```
/home/gryph/.minetest/mods/irc_commands/init.lua:47: attempt to index a nil value
stack traceback:
        /home/gryph/.minetest/mods/irc_commands/init.lua:47: in function 'func'
        /home/gryph/.minetest/mods/irc/botcmds.lua:55: in function 'bot_command'
        /home/gryph/.minetest/mods/irc/hooks.lua:149: in function 'f'
        /home/gryph/.minetest/mods/irc/irc/main.lua:60: in function 'invoke'
        /home/gryph/.minetest/mods/irc/hooks.lua:64: in function 'f'
        /home/gryph/.minetest/mods/irc/irc/main.lua:60: in function 'invoke'
        /home/gryph/.minetest/mods/irc/irc/main.lua:195: in function 'handle'
        /home/gryph/.minetest/mods/irc/irc/main.lua:165: in function 'think'
        /home/gryph/.minetest/mods/irc/init.lua:70: in function </home/gryph/.minetest/mods/irc/init.lua:70>
        [C]: in function 'xpcall'
        /home/gryph/.minetest/mods/irc/init.lua:70: in function </home/gryph/.minetest/mods/irc/init.lua:59>
        /usr/local/share/minetest/builtin/game/register.lua:348: in function </usr/local/share/minetest/builtin/game/register.lua:336>
```

**Comment by ShadowNinja** [comment]
Is the server in its channel?  That's a requirement, although it probably shouldn't crash if it isn't in the channel.

**Comment by fyregryph** [comment]
Yes, the server was in it's channel, and strangely it doesn't crash even
though it shows a stacktrace. From a user's point of view the extra
commands simply do nothing.

**Comment by deezl** [comment]
this issue is not resolved apparently. I am getting a similar problem, server doesn't crash, but we get this error when trying to login from IRC:

>...bin/../games/bluelobster_game/mods/irc_commands/init.lua:47: attempt to index a nil value
stack traceback:
        ...bin/../games/bluelobster_game/mods/irc_commands/init.lua:47: in function 'func'
        ...etest/bin/../games/bluelobster_game/mods/irc/botcmds.lua:55: in function 'bot_command'
        ...inetest/bin/../games/bluelobster_game/mods/irc/hooks.lua:155: in function 'f'
        ...test/bin/../games/bluelobster_game/mods/irc/irc/main.lua:60: in function 'invoke'
        ...inetest/bin/../games/bluelobster_game/mods/irc/hooks.lua:64: in function 'f'
        ...test/bin/../games/bluelobster_game/mods/irc/irc/main.lua:60: in function 'invoke'
        ...test/bin/../games/bluelobster_game/mods/irc/irc/main.lua:195: in function 'handle'
        ...test/bin/../games/bluelobster_game/mods/irc/irc/main.lua:165: in function 'think'
        ...minetest/bin/../games/bluelobster_game/mods/irc/init.lua:70: in function <...minetest/bin/../games/bluelobster_game/mods/irc/init.lua:70>
        [C]: in function 'xpcall'
        ...minetest/bin/../games/bluelobster_game/mods/irc/init.lua:70: in function <...minetest/bin/../games/bluelobster_game/mods/irc/init.lua:59>
        /home/chaoswormz/minetest/bin/../builtin/game/register.lua:341: in function </home/chaoswormz>> >/minetest/bin/../builtin/game/register.lua:329>
